h2o-py/h2o/backend/connection.py

```diff
@@ -815,6 +815,7 @@ def _process_response(response, save_to):
 
         # Client errors (400 = "Bad Request", 404 = "Not Found", 412 = "Precondition Failed")
         if status_code in {400, 404, 412} and isinstance(data, H2OErrorV3):
```

Comment:
btw, could be 
```if 400 <= status_code < 500 and …```
don't see the reason why this is limited to specific client errors

Reply:
I used the stacktrace of 401 once to determine what Jetty version is the user running :)

That is a weird case of course - but generally the stackrace for Unauthorized is helpful in validating that the message is actually generated by H2O (jetty) and not something along the way.

IMHO other 4xx than the listed ones are not common so we don't have to worry about how well they are printed - I would keep as is - definitely for 401 it can save time.